python/nano/notebooks/pytorch/cifar10/nano-trainer-example.ipynb

```diff
@@ -224,7 +224,8 @@
     "BATCH_SIZE = 32\n",
     "NUM_WORKERS = 0\n",
     "data_module = prepare_data(PATH_DATASETS, BATCH_SIZE, NUM_WORKERS)\n",
-    "EPOCHS = int(os.environ.get('FIT_EPOCHS', 30))"
+    "MAX_STEPS = int(os.environ.get('MAX_STEPS', -1))\n",
+    "EPOCHS = 30"
```

Comment:
which of "MAX_STEPS" and "EPOCHS" take effects?

Reply:
MAX_STEPS take effects.
One epoch contains more than 1500 steps in the pytorch cifar10 notebook. MAX_STEPS is set to 10 when unit testing.

Reply:
Now the "LIMIT_BATCHES" control how much of training dataset to check and "EPOCHS" effects the number of epochs.